VirtoCommerce.Storefront/Domain/Catalog/CatalogConverter.cs

```diff
@@ -432,6 +432,9 @@ public static Product ToProduct(this catalogDto.CatalogProduct productDto, Langu
                 result.Description = (result.Descriptions.FirstOrDefault(x => x.ReviewType.EqualsInvariant("FullReview")) ?? result.Descriptions.FirstOrDefault())?.Value;
             }
 
+            result.CanBeOrderedSeparately = bool.Parse(
+                result.Properties?.FirstOrDefault(x => x.Name == "Can_be_ordered_separately").Value);
```

Comment:
You should check the string for Null and an empty value and use the default value for the bool type in this case